lib/tarantool_server.py

```diff
@@ -829,14 +831,14 @@ def cleanup(self, *args, **kwargs):
                                              *args, **kwargs)
 
     def start(self, silent=True, wait=True, wait_load=True, rais=True, args=[],
-              **kwargs):
+              server_start_timeout=90, **kwargs):
```

Comment:
If you'll agree with my proposal about calculating a deadline inside the `start()` method and passing it down to `wait_until_started()` and `wait_seek()`, this point will not be actual. However I'll note that the `server_start_timeout` name looks redundant in the `start()` of a `Server` subclass: just `timeout` would be enough. Now different names are used for the same timeout and it is a bit confusing I think.

Reply:
Fixed